openehr-rm/src/main/java/com/nedap/archie/rm/support/identification/TerminologyId.java

```diff
@@ -9,7 +11,7 @@
  */
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "TERMINOLOGY_ID")
-public class TerminologyId {
+public class TerminologyId extends RMObject {
```

Comment:
According to the [specification](https://www.openehr.org/releases/BASE/latest/docs/base_types/base_types.html#_terminology_id_class) it should extend ObjectId.